### DISCUSSION TITLE
【投票】是否有必要保留web UI

### DISCUSSION THREAD

**Discussion by URenko** [URenko/Accesser#110] [Polls]
最初设计web UI是为了方便小白操作。然而在实际使用中，web UI似乎又用处不大？并且维护麻烦。

在v0.7.0中，我们暂时默认关闭了web UI（并且实际上是坏的）。现在考虑移除web UI，改为控制台，并且计划将配置文件从json改为ini或toml格式。

欢迎投票或发表建议

**Comment by URenko** [comment]
若维持web UI，则不改用ini或toml格式，原因是：
使用ini/toml是为了便于人类读写，因此会加入注释，但web UI修改后dump时无法优雅地保持这些注释。

**Comment by lxfly2000** [comment]
建议保留，至少我是一直在用的，比如我突然想把它关掉我就去网页点一下关闭，设置局域网访问也要用，如果JSON加不了注释完全可以复制一份样本作说明，我觉得用这个工具的人多少都是对JSON有了解的

**Comment by URenko** [comment]
https://github.com/URenko/Accesser/releases/tag/v0.8.0rc1
web UI将于v0.8.0移除，但保留部分HTTP API：
- `/pac/`
- `/shutdown`（这一功能过去实际上是坏的）
- `/CERT/root.*`（过去仅为`/CERT/root.crt`）

修改配置的HTTP API（如开启局域网访问）因涉及写入配置文件将不再可用
